doc/releases/changelog-dev.md

```diff
@@ -238,6 +238,8 @@
 
 <h3>Bug fixes 🐛</h3>
 
+* Fixes the capture of `QSVT` so that it no longer treats abstract values as metadata.
```

Comment:
```suggestion
* Fixes the capture of `QSVT` so that it no longer treats abstract values as metadata.
  [(#7360)](https://github.com/PennyLaneAI/pennylane/pull/7360)
```

Reply:
@albi3ro was this fix made purely for compatibility with JAX>0.4.28?

Reply:
```suggestion
* Fixed a bug that occurred with `qml.QSVT` and program capture enabled (`qml.capture.enable`), where abstract values were being treated as metadata.
```

Reply:
```suggestion
* With program capture enabled (`qml.capture.enable()`), `QSVT` no treats abstract values as metadata.
  [(#7360)](https://github.com/PennyLaneAI/pennylane/pull/7360)
```